justfile

```
@@ -2,7 +2,7 @@
     just --list --unsorted
 
 # Run all build-related recipes in the justfile
-run-all: install-deps format-python check-python check-unused test-python check-security check-spelling check-commits build-website
+run-all: install-deps format-python check-python type-check-python check-unused test-python check-security check-spelling check-commits build-website
```

```suggestion
run-all: install-deps format-python check-python check-unused test-python check-security check-spelling check-commits build-website
```

Should be included within the  `just check-python`